### ISSUE TITLE
No module named 'tensorflow' (with naive environment in ubuntu 16.04)

### ISSUE THREAD

**Issue by whitecoffee** [jupyter/notebook#3451]
Here is thing, I installed tensorflow-gpu==1.5.0 and library it needed. 
I didn't try any virtual environment, just installed them in local user's lib path.
Though I work well with python3 in terminal, but get 
`ImportError: no module named 'tensorflow'` 
in jupyter notebook. What makes this happened? And how to fix it?

P.S: tensorlfow_path -- '/usr/local/lib/python3.5/dist-packages/tensorflow'
        jupyter notebook_path -- '~/.local/bin/jupyter-notebook'
        I installed jupyter notebook without Anaconda

**Comment by takluyver** [comment]
It sounds like you're trying an impor...

**Comment by takluyver** [comment]
(Sorry about the previous message, looks like Github saved replies are broken just now - I've copied and pasted it below)

It sounds like you're trying an import in two different installations of Python, or two different environments. Check `sys.executable` to see which Python and environment you're running in, and `sys.path` to see where it looks to import modules.

Python packages must be installed separately for each copy of Python you use, and if you are using virtualenvs or conda envs, packages must be installed into each environment where you need them. Either the package is not installed in one, or a different version of the package is installed.

To install packages to a particular Python installation or environment using pip, you can run it like this:

```shell
path/to/python -m pip install ...
# Replace path/to/python with the path of the Python executable (sys.executable)
```

If you are using conda environments, you can install packages from conda like this:

```shell
source activate myenv  # On Windows, just 'activate myenv' (no 'source')
conda install ...
```

For more detail, see [this blog post](http://takluyver.github.io/posts/i-cant-import-it.html).

[ This is a saved reply because I answer similar questions often - sorry if it doesn't exactly fit your case ]

**Comment by whitecoffee** [comment]
Thanks for your attention :+1: . 
I followed your advice and here is my `sys.executable`
`/usr/bin/python3`

Then I enter following code at your command
`/usr/bin/python3 -m pip install jupyter`
Here is what I got (for the first two lines)
`Requirement already satisfied: jupyter in /usr/local/lib/python3.5/dist-packages
Requirement already satisfied: ipykernel in ./.local/lib/python3.5/site-packages (from jupyter)
`

The fact is that I install jupyter in a virtual env once. But now I need it now in global env which means in terminal command. The code seems to indicate that system combines the global env & virtual env. And the jupyter-notebook I used is in virtual env. 
What should I do now?  Can I install the jupyter-notebook in global lib? Or should I make some trade-off?
